data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/oidc/TokenProvider.java

```diff
@@ -23,22 +25,48 @@
 import io.fabric8.kubernetes.client.KubernetesClient;
 import io.fabric8.kubernetes.client.KubernetesClientBuilder;
 
+import java.util.concurrent.TimeUnit;
+
 public class TokenProvider {
 
+    private static final long TOKEN_EXPIRATION_SECONDS = 3600L; // 1 hour
+    private static final long EXPIRATION_BUFFER_TIME_SECONDS = 300L; // 5 minutes
+    private static final long CACHE_EXPIRATION_TIME =
```

Comment:
```suggestion
    private static final long CACHE_EXPIRATION_TIME_SECONDS =
```
nit: to align with the other names.